README.md

```diff
@@ -232,6 +232,18 @@ The program crashes and it has something to do with the Scheduler the extension
 > To display errors and to generate the code visualization, the extension calls the [Scheduler](https://github.com/iml130/pfdl). The Scheduler is still under development, so there may be bugs. If the extension crashes because of the Scheduler please open an issue in that repo.
 
 ## Release Notes
+For each release, a `.vsix` installation file for VS Code is provided, as well as a `bundle.js` source code file, containing the code that is responsible for the code visualization.
+
+Additionally, a license file is generated using the [OSS Review Toolkit (ORT)](http://oss-review-toolkit.org/ort/) to automatically conclude licenses and copyright information declared in the dependencies of this project (excluding `devDependencies` for npm). The file can be reproduced by following the [ORT installation guide](http://oss-review-toolkit.org/ort/docs/getting-started/installation) and running the following commands from inside the downloaded ORT repository:
```

Comment:
is it conclude or include?

Reply:
I agree that conclude is not the correct word here, changed it to "collect and summarize"

Reply:
Alright LFTM now :)